modules/javafx.base/src/test/java/test/javafx/util/converter/DateTimeStringConverterTest.java

```diff
@@ -62,127 +63,90 @@ public class DateTimeStringConverterTest {
         VALID_DATE_WITHOUT_SECONDS = c.getTime();
     }
 
-    private static Collection implementations() {
-        return Arrays.asList(new Object[][] {
-                { new DateTimeStringConverter(),
+    private record TestCase(DateTimeStringConverter converter, Locale locale, int dateStyle, int timetyle, String pattern,
+            DateFormat dateFormat, Date validDate) {}
+
+    private static Collection<TestCase> implementations() {
+        return List.of(
+                new TestCase(new DateTimeStringConverter(),
                         Locale.getDefault(Locale.Category.FORMAT), DateFormat.DEFAULT, DateFormat.DEFAULT,
-                        VALID_DATE_WITH_SECONDS, null, null },
+                        null, null, VALID_DATE_WITH_SECONDS),
```

Comment:
Why not put the last closing bracket on a new line aligned with `new` just like everyone always does with the curly braces?  Much more readable, and doesn't require the empty line between blocks to "make" it readable. So:

```java
        return List.of(
                new TestCase(new DateTimeStringConverter(),
                        Locale.getDefault(Locale.Category.FORMAT), DateFormat.DEFAULT, DateFormat.DEFAULT,
                        null, null, VALID_DATE_WITH_SECONDS
                ),
                // add more here...                        
        );     
```